python/copclib/mp/transform.py

```
@@ -86,6 +87,11 @@ def transform_multithreaded(
     if transform_function is None:
         transform_function = _copy_points_transform
 
+    if isinstance(writer, copc.FileWriter):
+        writer_header = writer.copc_config.las_header
+    elif isinstance(writer, copc.LazWriter):
+        writer_header = writer.laz_config.las_header
```

Should we raise an exception on `else` clause?